roofit/hs3/inc/RooFitHS3/RooJSONFactoryWSTool.h

```diff
@@ -159,11 +217,18 @@ class RooJSONFactoryWSTool {
    static void clearExportKeys();
    static void printExportKeys();
 
+   void importAllNodes(const RooFit::Detail::JSONNode &n);
+
    void importFunctions(const RooFit::Detail::JSONNode &n);
    void importPdfs(const RooFit::Detail::JSONNode &n);
    void importVariables(const RooFit::Detail::JSONNode &n);
+   void importFunction(const RooFit::Detail::JSONNode &n, bool isPdf);
+   void importVariable(const RooFit::Detail::JSONNode &n);
+   void configureVariable(const RooFit::Detail::JSONNode &p, RooRealVar &v);
    void importDependants(const RooFit::Detail::JSONNode &n);
 
+   void configureToplevelPdf(const RooFit::Detail::JSONNode &n, RooAbsPdf *pdf);
```

Comment:
```suggestion
   void configureToplevelPdf(const RooFit::Detail::JSONNode &n, RooAbsPdf &pdf);
```
Does this need to be a pointer? You should always prefer references in the function signatures if the `nullptr` case is not meaningful.